echo_handlers.go

```diff
@@ -16,14 +21,37 @@ import (
 	ddEcho "gopkg.in/DataDog/dd-trace-go.v1/contrib/labstack/echo.v4"
 )
 
+//go:embed public/*
+var embeddedAssets embed.FS
+
+//go:embed views/*
+var embeddedViews embed.FS
+
+type TemplateRegistry struct {
+	templates map[string]*template.Template
+}
+
+// Implement e.Renderer interface
+func (t *TemplateRegistry) Render(w io.Writer, name string, data interface{}, c echo.Context) error {
+	tmpl, ok := t.templates[name]
+	if !ok {
+		err := errors.New("Template not found -> " + name)
+		return err
+	}
+	return tmpl.ExecuteTemplate(w, "layout.html", data)
+}
+
 func (svc *Service) RegisterSharedRoutes(e *echo.Echo) {
 
 	templates := make(map[string]*template.Template)
 	templates["apps/index.html"] = template.Must(template.ParseFS(embeddedViews, "views/apps/index.html", "views/layout.html"))
 	templates["apps/new.html"] = template.Must(template.ParseFS(embeddedViews, "views/apps/new.html", "views/layout.html"))
 	templates["apps/show.html"] = template.Must(template.ParseFS(embeddedViews, "views/apps/show.html", "views/layout.html"))
 	templates["apps/create.html"] = template.Must(template.ParseFS(embeddedViews, "views/apps/create.html", "views/layout.html"))
-	templates["index.html"] = template.Must(template.ParseFS(embeddedViews, "views/index.html", "views/layout.html"))
+	templates["apps/new_with_pubkey.html"] = template.Must(template.ParseFS(embeddedViews, "views/apps/new_with_pubkey.html", "views/layout.html"))
+	templates["alby/index.html"] = template.Must(template.ParseFS(embeddedViews, "views/backends/alby/index.html", "views/layout.html"))
+	templates["about.html"] = template.Must(template.ParseFS(embeddedViews, "views/about.html", "views/layout.html"))
+	templates["lnd/index.html"] = template.Must(template.ParseFS(embeddedViews, "views/backends/lnd/index.html", "views/layout.html"))
```

Comment:
How do you get in this screen? Wasn't able to do that because there is always a user if we use the self-hosted version.

Reply:
currently you don't get to this screen because index redirects to /apps with the hard coded user. 
but I valued the consistency of having the same concept that's why I added the template. 
And I am still wondering if we should require a password for LND users can umbrel users access the app always only from the local network (and it's protected by that)?

I can remove it if you want.

Reply:
No, it's ok.